pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -427,6 +424,11 @@ def _detect_replacable_format_call(self, node: nodes.Const) -> None:
             elif isinstance(inferred_right, nodes.List):
                 if len(inferred_right.elts) > 1:
                     return
+            # If the inferred value is a tuple and the value comes
```

Comment:
This part is the actual logic change. Everything is just using variables so we don't have to call `\.parent` over and over and have ugly line breaks.